conans/client/runner.py

```diff
@@ -100,3 +102,27 @@ def _simple_os_call(self, command, cwd):
             finally:
                 os.chdir(old_dir)
             return result
+
+
+@contextmanager
+def pyinstaller_bundle_env_cleaned():
+    """Removes the pyinstaller bundle directory from LD_LIBRARY_PATH
+
+    :return: None
+    """
+    is_frozen = getattr(sys, 'frozen', False)
+
+    if not is_frozen or 'LD_LIBRARY_PATH' not in os.environ:
+        yield
+        return
+
+    # http://pyinstaller.readthedocs.io/en/stable/runtime-information.html#ld-library-path-libpath-considerations
+    ld_library_path = os.environ['LD_LIBRARY_PATH']
+    try:
+        os.environ['LD_LIBRARY_PATH'] = os.environ['LD_LIBRARY_PATH_ORIG']
+    except KeyError:
+        del os.environ['LD_LIBRARY_PATH']
```

Comment:
Why would you remove LD_LIBRARY_PATH in this case?

Reply:
this handles the case, when the variable LD_LIBRARY_PATH was not present before.
the pyinstaller bootstrap then creates it for its own purposes, but not LD_LIBRARY_PATH_ORIG since it would be empty and that has different meaning then not set.

sys.frozen==True and no LD_LIBRARY_PATH_ORIG implies there was no LD_LIBRARY_PATH before